roles/ncn-common-setup/tasks/gcp.yml

```diff
@@ -8,6 +8,22 @@
     owner: root
     group: root
 
+- name: backup /etc/sysconfig/network/config
+  copy:
+    src: /etc/sysconfig/network/config
+    dest: /etc/sysconfig/network/config.backup
```

Comment:
Is this restored later? I thought (and maybe I'm misremembering) this was put back at the end of the build.